src/App.scss

```diff
@@ -0,0 +1,298 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;700&display=swap');
+@import './reset.css';
+
+html,
+body {
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
+
+body {
+  background-color: #fafafa;
+}
+
+.app {
+  &.line {
+    height: 2px;
+    margin: 40px 100px 29px 100px;
+    border: solid 1px #e8e8e8;
+  }
+
+  &.main {
+    width: 100%;
+    margin: 0 auto;
+  }
+}
+
+.navbar {
+  width: 100%;
+  height: 95px;
+  background-color: #fff;
+  margin: 0 auto;
+
+  &.content {
+    max-width: 1100px;
+    border-radius: 4px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    margin: 0 0 0 100px;
+  }
+
+  span {
+    float: right;
+    margin-left: 28rem;
+    color: #0290ff;
+  }
+
+  &.content .count {
+    height: 30px;
+    width: 140px;
+  }
+
+  h1,
+  h2,
+  h3 {
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    padding-right: 15px;
+  }
+
+  h1 {
+    font-size: 30px;
+    font-weight: bold;
+    color: #0290ff;
+  }
+
+  h3 {
+    font-size: 13px;
+    font-stretch: normal;
+    letter-spacing: 1.9px;
+    color: #121212;
+  }
+}
+
+.book {
+  &.main-container {
+    width: 50%;
+  }
+
+  &.main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0 27px;
+    height: 170px;
+    margin: 37px 100px 0;
+    border-radius: 4px;
+    border: solid 1px #e8e8e8;
+    background-color: #fff;
+  }
+
+  &.title {
+    height: 29px;
+    margin: 3px 323px 0 0;
+    font-size: 22px;
+    font-family: 'Roboto Slab', serif;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
```

Comment:
- [ ] [REQUIRED] Please, make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule